examples/datalog.lisp

```diff
@@ -0,0 +1,23 @@
+(begin
```

Comment:
begin (here and on line 10) are no longer needed in the file, since multiline parse is now supported. 
The go example still needs it as it evals a single line each time